sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -115,7 +113,7 @@ public DigitalTwinsServiceVersion getServiceVersion() {
     @ServiceMethod(returns = ReturnType.SINGLE)
     public <T> Mono<T> createDigitalTwin(String digitalTwinId, T digitalTwin, Class<T> clazz)
     {
-        return createDigitalTwinWithResponse(digitalTwinId, digitalTwin, clazz)
+        return createDigitalTwinWithResponse(digitalTwinId, digitalTwin, clazz, null)
```

Comment:
so we are defaulting options to null? is there a scenario where we would want these options to have a default value instead?

Reply:
Since they are optional, it is always allowed. It saves us from instantiating new options classes on each call. I didn't see any particular cases where it made more sense to create a default options object instead.

Reply:
If "null" is the expected value, then yes, I like leaving it as is. I was thinking of the scenario where we have some request Id in options, which we would want to have a default value, rather than null. But if that is not relevant for our API calls, then what we have is good.